assets/js/highscores.js

```diff
@@ -0,0 +1,53 @@
+// global
+const main = document.getElementById("main-content");
+
+const highScoresSection = document.getElementById("high-scores-section");
+
+// getScores from local storage
+const readFromLocalStorage = () => {
+  // get from LS by key
+  const getScores = localStorage.getItem("highscores");
+  // parse LS data
+  const parsedData = JSON.parse(getScores);
```

Comment:
Let's return directly here, rather than declaring a variable that we don't really need:

`return JSON.parse(getScores)`